src/main/java/org/cactoos/scalar/Ternary.java

```diff
@@ -39,10 +39,10 @@
  *
  * <pre>{@code
  * new Ternary<>(
- *     5,
+ *     someScalarOfInteger,
```

Comment:
@victornoel Camel-cased variables are against the current code style, probably we should not have them in examples too.

Reply:
@andreoss right, I also simplified the example